### PR TITLE
WEBAPP-558: Deliver malte

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/). 
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **WEBAPP**.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -8,60 +8,54 @@ parameters:
...
 
 workflows:
   version: 2
-  commit: # Builds every commit. Delivers to webnext on develop.
+  commit: # Builds every commit. Delivers to webnext.integreat.app and webnext.malteapp.de on develop.
     unless: << pipeline.parameters.api_triggered >>
     jobs:
-      - build:
-          build_config: "integreat-test-cms"
+      - test_and_build
```
Comment (klinzo) [inline .circleci/config.yml:21]: Not sure about this naming change

Comment (maxammann) [reply .circleci/config.yml:21]: I think build is enough, as it is not only test but also linting etc.

File: .circleci/trigger-pipeline [hunk 2]
```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
 branch="develop"  # Default branch
-api_token="$CIRCLE_API_USER_TOKEN"  # Default to $CIRCLE_API_USER_TOKEN
-build_config="integreat-test-cms" # Use "integreat-test-cms" config by default (delivery to webnext)
+api-token="$CIRCLE_API_USER_TOKEN"  # Default to $CIRCLE_API_USER_TOKEN
+production-delivery="false" # Do not deliver to production by default
```
Comment (maxammann) [inline .circleci/trigger-pipeline:5]: Sorry! should be _

File: build-configs/themes/integreat/index.js [hunk 3]
```diff
@@ -14,8 +14,8 @@ const theme: ThemeType = {
 }
 
 export const darkTheme: ThemeType = {
-  colors: integreatDarkColors,
-  ...theme
+  ...theme,
+  colors: integreatDarkColors
 }
```
Comment (Schedulaar) [inline build-configs/themes/integreat/index.js:19]: wat? :laughing: Didn't this already work? :thinking:

File: package.json [hunk 4]
```diff
@@ -97,7 +97,7 @@
     "@integreat-app/eslint-config-node": "^0.6.0",
     "@integreat-app/eslint-config-react": "^0.6.1",
     "@octokit/auth-app": "^2.4.5",
-    "@octokit/rest": "^17.9.0",
+    "@octokit/rest": "^17.11.0",
```
Comment (Schedulaar) [inline package.json:100]: What is this used for?

File: build-configs/themes/integreat/index.js (hunk 3 above)
Comment (klinzo) [reply build-configs/themes/integreat/index.js:19]: I think it did work but my changes and moving this to separate build configs broke it. Working now again though.

**Pull request merged**